doc/how_to/load_matalb_data.rst

```diff
@@ -0,0 +1,66 @@
+Exporting MATLAB Data to Binary & Loading in SpikeInterface
+===========================================================
+
+In this tutorial, we'll go through the process of exporting your data from MATLAB in a binary format and then loading it using SpikeInterface in Python. Let's break down the steps.
+
+Exporting Data from MATLAB
+--------------------------
+
+First, ensure your data is structured correctly. The data matrix should be organized such that the first dimension corresponds to samples/time and the second dimension to channels.
+
+.. code-block:: matlab
+
+   % Define the size of your data
+   num_samples = 1000;
+   num_channels = 384;
+
+   % Generate random data as an example
+   data = rand(num_samples, num_channels);
+
+   % Write the data to a binary file
+   fileID = fopen('your_data_as_a_binary.bin', 'wb');
+   fwrite(fileID, data, 'double');
+   fclose(fileID);
+
+.. note::
+
+   In a real-world scenario, replace the random data generation with your actual data.
+
+Loading Data in SpikeInterface
+-----------------------------
+
+This should produce a binary file called `your_data_as_a_binary.bin` in your current MATLAB directory.
+You will need the complete path (i.e. its location on your computer) to load it in Python.
+
+Once you have your data in a binary format, you can seamlessly load it into SpikeInterface using the following script:
+
+.. code-block:: python
+
+   from spikeinterface.core.binaryrecordingextractor import BinaryRecordingExtractor
+   from pathlib import Path
+
+   # Define the path to your binary file
+   file_path = Path("/The/Path/To/Your/Data/your_data_as_a_binary.bin")
+
```

Comment:
```suggestion
   file_path = Path("/The/Path/To/Your/Data/your_data_as_a_binary.bin")
   # or for Windows
   # file_path = Path(r"c:\path\to\your\data\your_data_as_a_binary.bin")
```
I tend to recommend just warning windows users ahead of time so that they don't come with a bunch of path issues.

Reply:
Good point.
What do you think of the "path/to/your/data/" visual trick to let them know that it is a path? In another package we have similar tutorials:

https://neuroconv.readthedocs.io/en/main/conversion_examples_gallery/recording/spikeglx.html

And we opted for something more concise there (just a variable in caps). I would like to hear if you have any ideas on how to make this clearer.

Reply:
I think the path to variable is super tricky. Because some users will actually put `path/to/variable` thinking that it is a hard coded path. And then adding in windows confusion with the opposite slashes. But as far as strategies I still think it is best despite being a bit longer and less concise.

Only issue with the concise variable style is that for other users they might not realize what's behind the variable. So that's why I prefer the `path/to/variable` style although it is not perfect. For Linux users I think they tend to be more comfortable with the path notation. But Windows has a right+click copy path option that will copy with `x\y\z` instead, so even though they could put in a Path(`c:/my/data`) they almost never do. And then since Windows has backslashes with escapes if we don't warn them about using the raw string instead they will get pathway errors due to the escaping and get frustrated.

In summary, I would leave it as you have it, but add in a the comment for windows users so when they copy the path from the computer they know that they likely need the r.

Reply:
Thanks for your input.